frontend/src/pages/Workflows/actions.js

```diff
@@ -268,6 +274,13 @@ export function hideWorkflowitemAdditionalData() {
   };
 }
 
+export function showReasonDialog(isRejectReasonDialogShown){
+  return {
+    type: SHOW_REASON_DIALOG,
+    isRejectReasonDialogShown
+  };
+}
```

Comment:
Make two functions without paramater list: `showReasonDialog` and `hideReasonDialog` . Your version is too confusing